qubes/api/internal.py

```diff
@@ -361,3 +361,65 @@ async def suspend_post(self):
                         qubes.config.suspend_timeout,
                         "qubes.SuspendPostAll",
                     )
+
+    @qubes.api.method(
+        "internal.dispvm.preload.Create", scope="local", write=True
+    )
+    async def dispvm_preload_create(self):
+        """
+        Method called to create a preloaded DispVM.
+
+        :return:
+        """
+        dispvm_template = self.arg
+        preload_dispvm = dispvm_template.features.get("preload-dispvm", None)
+        ## TODO: should this function receive disposable name as argument
+        ##       or call admin.vm.CreateDisposable?
```

Comment:
IMO makes more sense to create here.

There should be also somewhere a counter how many preloaded disposables should be and if that limit (which may be 0) is reached already, do nothing here.